nsacyber/HIRS
# [#844] Update HIRS/Utils For Checkstyle

### Description:
Fix all the checkstyle issues in all the files inside the HIRS/UTILS module.

### Testing Instructions:
To ensure that checkstyle is enabled in your IDE, follow the following steps:
1. If you are using Intellij as your preferred IDE, open up the IDE, go to files (top left corner of the window), click on settings and search for the word **plugins**.  Look for the plugin CheckStyle-IDEA and enable that plugin for your IDE.
2. Once that's done, go to search box on the settings window and write **code style**. There should be a drop downlist and a gear icon next to the word **Scheme**. Click on the gear and click on import scheme. Import checkstyle configuration.
3. Once you've imported the plugin and you've imported the XML formatter file. Go back up to the search box in your settings window and search for **Actions on Save** and check **reformat code**, **optimize imports**, **rearrange code** and **run code cleanup**. Hit apply once you've checked all those actions. Now your code will be "cleaned" every time you hit save.

To verify that this ticket has been completed, run this command in your IDEA terminal or in a terminal window:
`./gradlew :HIRS_Utils:check --rerun-tasks`

There should be no checkstyle errors once this command has been run.
